src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt

```diff
@@ -25,7 +25,7 @@ data class Offender(
   @NotNull
   val prisonerNumber: String,
 
-  val prisonId: String,
+  var prisonId: String,
```

Comment:
Does this need to be var? If you need to update it for an existing offender, could you instead do

val updatedOffender = offender.copy(prisonId = newPrisonId)